packages/model-viewer/src/three-components/Model.ts

```diff
@@ -157,6 +157,7 @@ export default class Model extends Object3D {
   set animationTime(value: number) {
     if (this.currentAnimationAction != null) {
       this.currentAnimationAction.time = value;
```

Comment:
I believe you can remove this line and the corresponding null check, since I think the method you added does this internally. Though, please test it to make sure I'm right :)